satpy/tests/reader_tests/test_geocat.py

```diff
@@ -96,7 +96,7 @@ def get_test_content(self, filename, filename_info, filetype_info):
                     if key + '/attr/' + a in file_content:
                         attrs[a] = file_content[key + '/attr/' + a]
                 if val.ndim > 1:
-                    file_content[key] = DataArray(val, dims=('y', 'x'), attrs=attrs)
+                    file_content[key] = DataArray(val, dims=('lines', 'elements'), attrs=attrs)
```

Comment:
why not x and y ?

Reply:
The test is meant to represent what is in the file. The NetCDF4 utility wrapper does not rename any dimensions so in the geocat files the dimensions are actually lines and elements.